pkg/util/httpstream/spdy/roundtripper.go

```diff
@@ -87,6 +87,11 @@ func (s *SpdyRoundTripper) dial(req *http.Request) (net.Conn, error) {
 		return nil, err
 	}
 
+	// Return if we were configured to skip validation
```

Comment:
we should honor the tlsConfig.InsecureSkipVerify setting on the transport by skipping the hostname verify